interfaces.go

```diff
@@ -238,3 +238,13 @@ type StateSyncFilter struct {
 	ID       uint64
 	Contract common.Address
 }
+
+// interface for whitelist service
+type ChainValidator interface {
```

Comment:
Why do we need 2 practically the same interfaces?

Reply:
Do you mean here and in `checker.go`?